components/gitpod-protocol/src/messaging/error.ts

```diff
@@ -37,41 +36,18 @@ export namespace ApplicationError {
             throw e;
         }
     }
-
-    export function fromGRPCError(e: any, data?: any): ApplicationError {
```

Comment:
we should never mindlessly pump error like that, we should do proper translation and if we cannot translate use INTERNAL to iterate on it